test/game/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+find_package(Boost ${MINIMUM_BOOST_VERSION}
+    COMPONENTS
+        unit_test_framework
+        log
+        log_setup
+    REQUIRED)
+
+add_executable(fo_unittest_game
```

Comment:
This is not a unit test, but rather a smoke and system test.

Can you please rename the whole `test/game` directory to `test/system` and the executable to `fo_smoketest_game`?

Reply:
Renamed.